Nodejs/Product/Nodejs/Commands/DiagnosticsCommand.cs

```diff
@@ -15,25 +15,31 @@
 //*********************************************************//
 
 using System;
-using System.Globalization;
 using System.IO;
 using System.Linq;
 using System.Reflection;
 using System.Text;
 using System.Threading;
-using Microsoft.NodejsTools;
-using Microsoft.NodejsTools.Analysis;
 using Microsoft.NodejsTools.Logging;
 using Microsoft.VisualStudioTools;
 using Microsoft.VisualStudioTools.Project;
 
 namespace Microsoft.NodejsTools.Commands {
-    /// <summary>
-    /// Provides the command for starting a file or the start item of a project in the REPL window.
-    /// </summary>
     internal sealed class DiagnosticsCommand : Command {
```

Comment:
Tested with no NTVS project, one NTSV project, and multiple projects. Here's an example:

```
Use Ctrl-C to copy contents

Projects:
    Project: NodejsWebApp3\NodejsWebApp3.njsproj
        Kind: Node.js
        StartupFile: c:\users\matb.redmond\documents\visual studio 2015\Projects\NodejsWebApp3\NodejsWebApp3\server.js
        WorkingDirectory: .
        PublishUrl: <undefined>
        SearchPath: <undefined>
        CommandLineArguments: <undefined>
        Analysis Log: 
        Analysis loading...


Logged events/stats:
    Analysis Level: NodeLsNone
    SurveyNewsFrequency: Disabled
    Debug Launches: 0
    Normal Launches: 0


Loaded assemblies:
    Accessibility, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    Anonymously Hosted DynamicMethods Assembly, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null, FileVersion=(null)
    ClwWizard, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    ConcurrencyPlugins, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    ConcurrencyPlugins.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    CustomMarshalers, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    EntityFramework, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.2.11026.0
    EnvDTE, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    EnvDTE100, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    EnvDTE80, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    EnvDTE90, Version=9.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Esent.Interop, Version=1.9.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.9.2.0
    GuardMethodAssembly, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null, FileVersion=(null)
    LibGit2Sharp, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25029.0
    MetadataViewProxies_3eec589f-b134-439f-831b-45015d785ee9, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null, FileVersion=(null)
    MetadataViewProxies_61a168de-c17b-4920-a45f-b873750ac6b1, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null, FileVersion=(null)
    Microsoft.Activities.Design.Services, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.0.50408.2
    Microsoft.Alm.Shared, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.ApplicationInsights, Version=0.17.2.169, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=0.17.2.169
    Microsoft.ApplicationInsights.PersistenceChannel, Version=0.17.2.179, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=0.17.2.179
    Microsoft.ApplicationInsights.UniversalTelemetryChannel, Version=0.17.2.179, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=0.17.2.179
    Microsoft.AspNet.Mvc.Razor.Host, Version=6.0.0.0, Culture=neutral, PublicKeyToken=null, FileVersion=6.0.0.11009
    Microsoft.AspNet.Razor, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null, FileVersion=4.0.0.11009
    Microsoft.AspNet.Razor.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null, FileVersion=4.0.0.11009
    Microsoft.AspNet.Scaffolding.12.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.AspNet.Scaffolding.EntityFramework.12.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.AspNet.Scaffolding.Mvc.5.0, Version=5.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=5.2.40314.0
    Microsoft.AspNet.Scaffolding.Mvc6, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.AspNet.Scaffolding.VSExtension.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.AspNet.SignalR.Client, Version=2.1.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=2.1.30912.170
    Microsoft.AspNet.SignalR.Core, Version=2.1.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=2.1.30912.170
    Microsoft.Azure.ActiveDirectory.GraphClient, Version=2.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=2.1.0
    Microsoft.Azure.ActiveDirectory.GraphClient, Version=2.1.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=2.1.1
    Microsoft.Build, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.Build.Framework, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.Build.Tasks.Core, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.Build.Utilities.Core, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.Build.Utilities.v4.0, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    Microsoft.CodeAnalysis, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.CodeAnalysis.CSharp, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.CodeAnalysis.CSharp.EditorFeatures, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.CodeAnalysis.CSharp.Features, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.CodeAnalysis.CSharp.InteractiveEditorFeatures, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.CodeAnalysis.CSharp.Workspaces, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.CodeAnalysis.EditorFeatures, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.CodeAnalysis.EditorFeatures.Text, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.CodeAnalysis.Elfie, Version=0.10.6.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=0.10.6.0
    Microsoft.CodeAnalysis.Features, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.CodeAnalysis.InteractiveEditorFeatures, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.CodeAnalysis.InteractiveFeatures, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.CodeAnalysis.Scripting, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.CodeAnalysis.TypeScript.EditorFeatures, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.30615.1.0
    Microsoft.CodeAnalysis.VisualBasic, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.CodeAnalysis.VisualBasic.EditorFeatures, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.CodeAnalysis.VisualBasic.Features, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.CodeAnalysis.VisualBasic.Workspaces, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.CodeAnalysis.Workspaces, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.CodeAnalysis.Workspaces.Desktop, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.Cosmos.ScopeStudio, Version=2.0.4700.0, Culture=neutral, PublicKeyToken=74d8d83b5ffd5692, FileVersion=2.0.4700.0
    Microsoft.Cosmos.ScopeStudio.BusinessObjects.SymbolManager, Version=2.0.4700.0, Culture=neutral, PublicKeyToken=74d8d83b5ffd5692, FileVersion=2.0.4700.0
    Microsoft.Cosmos.ScopeStudio.VsExtension.EditorMarginExtension.Scope, Version=2.0.4700.0, Culture=neutral, PublicKeyToken=74d8d83b5ffd5692, FileVersion=2.0.4700.0
    Microsoft.CSharp, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    Microsoft.CSS.Core, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.CSS.Editor, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.Data.Edm, Version=5.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=5.0.0.50627
    Microsoft.Data.Entity.Design.BootstrapPackage, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.41103.0
    Microsoft.Data.Entity.Design.Package, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.41103.0
    Microsoft.Data.OData, Version=5.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=5.0.0.50627
    Microsoft.Data.Tools.Schema.Sql, Version=13.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=13.0.3314.1
    Microsoft.Diagnostics.MemoryGraph, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.Diagnostics.Tracing.EventSource, Version=1.1.16.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.1.16.0
    Microsoft.DiagnosticsHub.CpuSampling, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.DiagnosticsHub.DataWarehouse.Interop, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.DiagnosticsHub.Presentation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.DiagnosticsHub.Runtime, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.DiagnosticsHub.SDK, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.DiagnosticsHub.VisualStudio.Package, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.DiagnosticsHub.VisualStudio.SDK, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.GeneratedCode, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null, FileVersion=(null)
    Microsoft.GeneratedCode, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null, FileVersion=(null)
    Microsoft.Html.Core, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.Html.Editor, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.IdentityModel.Clients.ActiveDirectory, Version=2.16.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=2.16.20422.1202
    Microsoft.Internal.VisualStudio.UserNotifications, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.JavaScript.Services, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.JavaScript.Services.Implementation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.JSON.Core, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.JSON.Editor, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.LightSwitch.JavaScript.Project, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.MSXML, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.NodejsTools, Version=1.0.2015.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.2.4100.00
    Microsoft.NodejsTools.Analysis, Version=1.0.2015.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.2.4100.00
    Microsoft.NodejsTools.InteractiveWindow, Version=1.0.2015.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.2.4100.00
    Microsoft.NodejsTools.Npm, Version=1.0.2015.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.2.4100.00
    Microsoft.NodejsTools.Profiling, Version=1.0.2015.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.2.4100.00
    Microsoft.NodejsTools.ProjectWizard, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.2.4100.00
    Microsoft.NodejsTools.Telemetry.14.0, Version=14.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.1.0
    Microsoft.NodejsTools.TestAdapter, Version=1.0.2015.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.2.4100.00
    Microsoft.Office.SharePoint.Tools, Version=15.0.0.0, Culture=neutral, PublicKeyToken=71e9bce111e9429c, FileVersion=15.0.4420.1017
    Microsoft.ProjectK.Npm, Version=1.0.2014.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.0.4100.00
    Microsoft.SharePoint.Client.Runtime, Version=15.0.0.0, Culture=neutral, PublicKeyToken=71e9bce111e9429c, FileVersion=15.0.4561.1000
    Microsoft.SmartDevice.Connectivity, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25123.0
    Microsoft.SmartDevice.ConnectivityWrapper.11, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25123.0
    Microsoft.SmartDevice.ConnectivityWrapper.12, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25123.0
    Microsoft.SmartDevice.MultiTargeting.Connectivity, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25123.0
    Microsoft.SqlServer.ConnectionDlg.Core, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.60519.0
    Microsoft.SqlServer.RegSvrEnum, Version=13.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91, FileVersion=13.0.1601.5 ((SQL16_RTM).160429-2226)
    Microsoft.TeamFoundation.Build.Client, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.Build.Common, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.Build.Controls, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.Build.Controls.Common, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.Build.Workflow, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.Build2.WebApi, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.Client, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.CodeReview.Components, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.CodeReview.Controls, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.Common, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.Controls, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.Discussion.Client, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.Git.Controls, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.Git.CoreServices, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.Git.Provider, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.Git.SharedServices, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.ProjectManagement, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.SourceControl.WebApi, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.TeamExplorer.Navigation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.TestManagement.Client, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.VersionControl.Client, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.VersionControl.Common, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.VersionControl.Controls, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.VersionControl.Controls.Common, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.VersionControl.UIFeatures, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.WorkItemTracking.Client, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.WorkItemTracking.Common, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.WorkItemTracking.Controls, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.TeamFoundation.WorkItemTracking.ControlsCore, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.Threading.Tasks, Version=1.0.12.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.0.168.0
    Microsoft.Threading.Tasks.Extensions, Version=1.0.12.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.0.168.0
    Microsoft.VisualC.Editor.Implementation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualC.QuickSearch.Implementation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Activities, Version=14.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Activities.CSharp, Version=14.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.AppDesigner, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.ApplicationInsights, Version=7.0.20622.1, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=7.0.20622.1
    Microsoft.VisualStudio.ApplicationInsights.Interfaces, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.60211.1
    Microsoft.VisualStudio.AppResponsiveness, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.AppxPackage, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Authentication.ConnectedServices, Version=1.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.1.63009.34349
    Microsoft.VisualStudio.Azure.CommonAzureTools.Utilities, Version=1.7.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.7.40521.1
    Microsoft.VisualStudio.Azure.ResourceManagement, Version=1.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.1.63009.34349
    Microsoft.VisualStudio.Azure.Services.Utilities, Version=1.3.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.3.30604.1604
    Microsoft.VisualStudio.Azure.VSIntegration, Version=1.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.1.63009.34349
    Microsoft.VisualStudio.CallHierarchy.Package.Implementation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.ClientDiagnostics, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.ClientDiagnostics.Common, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.ClientDiagnostics.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.ClientDiagnostics.XamlDiagnostics, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.ClientDiagnostics.XamlDiagnostics.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.CodeAnalysis, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.CodeAnalysis.Sdk.UI, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.CodeAnalysis.VCPlugin, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.CodeStore.Internal, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.CommandBars, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.CommonIDE, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.ComponentModelHost, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.ComponentModelHost.Implementation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Composition, Version=14.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.3.11.26280
    Microsoft.VisualStudio.Composition.Configuration, Version=14.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.3.11.26280
    Microsoft.VisualStudio.ConnectedServices, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=2.0.63013.44418
    Microsoft.VisualStudio.ConnectedServices.Azure.Authentication.Common, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.0.50720.1
    Microsoft.VisualStudio.ConnectedServices.Azure.Storage, Version=1.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.1.63009.34349
    Microsoft.VisualStudio.ConnectedServices.Contracts, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.ConnectedServices.Package, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.ConnectedServices.Package2, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=2.0.63013.44418
    Microsoft.VisualStudio.ContextManagement.Package, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.VisualStudio.ContextManagement.Public, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.VisualStudio.CoreUtility, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.CppSvc.Internal, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.CSharp.Repl, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.VisualStudio.Data.Core, Version=9.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=9.0.21022.8
    Microsoft.VisualStudio.Data.Framework, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Data.Host, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Data.Interop, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.Data.Package, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Data.Services, Version=9.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=9.0.21022.8
    Microsoft.VisualStudio.Data.Tools.DataCompare, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.60519.0
    Microsoft.VisualStudio.Data.Tools.Delta, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.60519.0
    Microsoft.VisualStudio.Data.Tools.Delta.UI, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.60519.0
    Microsoft.VisualStudio.Data.Tools.Design.Common, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.60519.0
    Microsoft.VisualStudio.Data.Tools.Design.Core, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.60519.0
    Microsoft.VisualStudio.Data.Tools.Package, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.60519.0
    Microsoft.VisualStudio.Data.Tools.Package.SqlProfile, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.60519.0
    Microsoft.VisualStudio.Data.Tools.Relational.Design.Package, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.60519.0
    Microsoft.VisualStudio.Data.Tools.SqlDebugger, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.60519.0
    Microsoft.VisualStudio.Data.Tools.SqlEditor, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.60519.0
    Microsoft.VisualStudio.Data.Tools.SqlObjectExplorer, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.60519.0
    Microsoft.VisualStudio.Data.Tools.SqlPdwExtensions, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.60519.0
    Microsoft.VisualStudio.DataDesign.Common, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.DataDesign.DataObjectGenerators, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.DataDesign.DataSetDesigner, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.DataDesign.DataSourceProviders, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.DataDesign.Interfaces, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.DataDesign.WpfComponents, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.DataDesign.WpfDataTool, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Debugger.Engine, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Debugger.Interop.10.0, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.Debugger.Interop.11.0, Version=11.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.Debugger.Interop.12.0, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.Debugger.Interop.Internal, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.Debugger.InteropA, Version=9.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.Debugger.Parallel, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Debugger.Parallel.Extension, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Debugger.Parallel.Gpu, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Debugger.Parallel.Gpu.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Debugger.Parallel.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Debugger.Runtime, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.DebuggerVisualizers, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Design, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Designer.Interfaces, Version=1.0.5000.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.DesignTools.DesignerContract, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.DesignTools.DesignerHost, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.DesignTools.HostUtility, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.DesignTools.Markup, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.DesignTools.Utility, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.DesignTools.XamlDesignerHost, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.DesignTools.XamlDesignerHost.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.DesignTools.XamlLanguageService, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.DesignTools.XamlLanguageService.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.DeveloperTools, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.DeviceConnectivity.Interop.11.0, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.Diagnostics.Assert, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Diagnostics.Common, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Diagnostics.HubExtensions, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Diagnostics.Measurement, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Diagnostics.PerformanceProvider, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Diagrams.View, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Diagrams.View.Interfaces, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Dialogs, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Diff, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Editor, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Editor.Implementation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Editors, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Editors.WCF, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.ErrorListPkg, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.ExtensibilityHosting, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.ExtensionManager, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.ExtensionManager.Implementation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.ExtensionsExplorer, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.ExtensionsExplorer.UI, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.GotoCode, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.GotoCode.Git, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.GotoCode.Interfaces, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.GotoCode.TeamFoundation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.GraphDocumentPackage, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Graphics, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.GraphModel, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.GraphProviderPackage, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Html.Package, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.IDE.ToolboxControlsInstaller, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.ImageCatalog, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Imaging, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Imaging.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.InteractiveServices, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.VisualStudio.InteractiveWindow, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.VisualStudio.JavaScript.Project, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.JavaScript.Project.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.JavaScript.Web.Extensions, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.JSLS, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.JSLS.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.JSON.Package, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.Language.CallHierarchy, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Language.CallHierarchy.Implementation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Language.GenerateType, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Language.GenerateType.Implementation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Language.Intellisense, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Language.NavigateTo.Implementation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Language.NavigateTo.Interfaces, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Language.QuickSearch.FileNameProvider, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Language.StandardClassification, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.LanguageServices, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.VisualStudio.LanguageServices.CSharp, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.VisualStudio.LanguageServices.Implementation, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.VisualStudio.LanguageServices.SolutionExplorer, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.VisualStudio.LanguageServices.Telemetry, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.VisualStudio.LanguageServices.TypeScript, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.30615.1.0
    Microsoft.VisualStudio.LanguageServices.VisualBasic, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.VisualStudio.Licensing, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.ManagedInterfaces, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=8.0.50727.42
    Microsoft.VisualStudio.Modeling.Components.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Modeling.Sdk.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Modeling.Sdk.Diagrams.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Modeling.Sdk.Shell.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.MSTest.TestWindow, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Office365, Version=1.6.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.6.51113.1
    Microsoft.VisualStudio.OLE.Interop, Version=7.1.40304.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.OnlineLicensing, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.OnlineLicensing.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.ORDesigner.DslPackage, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Package.LanguageService.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.PackageManifestEditor, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.PerformanceTools.EditorIntegration, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Platform.AppDomainManager, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Platform.VSEditor, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Platform.VSEditor.Interop, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.Platform.WindowManagement, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Platform.WindowManagement.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.PowerProfiler, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25123.0
    Microsoft.VisualStudio.PowerShellTools.Package, Version=1.8.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.8.40521.1
    Microsoft.VisualStudio.ProductKeyDialog, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.ProductKeyDialog.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Progression.CodeSchema, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Progression.Common, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Progression.DataManager, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Progression.GraphControl, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Progression.Interfaces, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Progression.Native.CodeProvider, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Progression.VSProvider, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Progression.WpfCommon, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Project.VisualC.VCProjectEngine, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.ProjectAggregator, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.ProjectSystem.DNX.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.ProjectSystem.Implementation, Version=14.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.151.54133
    Microsoft.VisualStudio.ProjectSystem.Utilities.v14.0, Version=14.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.151.54133
    Microsoft.VisualStudio.ProjectSystem.V14Only, Version=14.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.151.54133
    Microsoft.VisualStudio.ProjectSystem.VS.Implementation, Version=14.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.151.54133
    Microsoft.VisualStudio.ProjectSystem.VS.V14Only, Version=14.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.151.54133
    Microsoft.VisualStudio.QualityTools.Common, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.QualityTools.ExecutionCommon, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.QualityTools.Vsip, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.ReferenceManager.Contracts, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.ReferenceManager.Dialog, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.ReferenceManager.Providers, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.ReferenceManager.Providers.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.RemoteControl, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.244.3504
    Microsoft.VisualStudio.ScriptedHost, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.ServerExplorer, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.ServiceModel, Version=14.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Services.Client, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.VisualStudio.Services.Common, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.VisualStudio.Services.Integration, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.VisualStudio.Services.WebApi, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.VisualStudio.SharePoint, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.AddServiceReference, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.AppDeployment, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.Bootstrapper, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.BusinessConnectivityServices, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.ControlsIntelliSense, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.Designers.BusinessDataConnectivity.DslPackage, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.Designers.List, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.Designers.Package, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.Explorer, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.Explorer.Extensions, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.Explorer.Extensions.Implementation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.Internal, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.Project, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.ProjectExtensions.Bdc, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.ProjectExtensions.VSPackage, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.ProjectExtensions.Wizards, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.Remote, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.Remote.Commands, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.SharePoint.WorkflowDesignerSupport, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23928.0
    Microsoft.VisualStudio.Shell.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Shell.Connected, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Shell.Connected.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Shell.Design, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Shell.Immutable.10.0, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=10.0.30319.1
    Microsoft.VisualStudio.Shell.Immutable.11.0, Version=11.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=11.0.50727.1
    Microsoft.VisualStudio.Shell.Immutable.12.0, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=12.0.21003.1
    Microsoft.VisualStudio.Shell.Immutable.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Shell.Interop, Version=7.1.40304.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.Shell.Interop.10.0, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.Shell.Interop.11.0, Version=11.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.Shell.Interop.12.0, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.Shell.Interop.8.0, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.Shell.Interop.9.0, Version=9.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.Shell.TreeNavigation.GraphProvider, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Shell.TreeNavigation.HierarchyProvider, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Shell.UI.Internal, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Shell.UI.Internal.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Shell.ViewManager, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Shell.VsHub.Immutable, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.TailoredProjectServices, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.TailoredProjectServices.Debugger, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.TailoredProjectServices.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.TaskRunnerExplorer, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.TaskRunnerExplorer.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.TaskRunnerExplorer.Extensions, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.TeamFoundation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.VisualStudio.TeamFoundation.Build, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.VisualStudio.TeamFoundation.Client, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.VisualStudio.TeamFoundation.SharePointReporting, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.VisualStudio.TeamFoundation.VersionControl, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.VisualStudio.TeamFoundation.WorkItemTracking, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.VisualStudio.TeamSystem.Common, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.98.25331.0
    Microsoft.VisualStudio.Telemetry, Version=14.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.674.58037
    Microsoft.VisualStudio.Telemetry.Package, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.TemplateProviders.Implementation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.TemplateWizard, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.TemplateWizardInterface, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=8.0.50727.42
    Microsoft.VisualStudio.TestPlatform.Client, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.TestPlatform.Common, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.TestPlatform.Core, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.TestPlatform.ObjectModel, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.TestPlatform.TestGeneration.Package, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.TestPlatform.Utilities, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.TestWindow, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.TestWindow.Core, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.TestWindow.CppUnitTestExtension, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.TestWindow.Interfaces, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.TestWindow.VSTest, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Text.Data, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Text.Internal, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Text.Logic, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Text.UI, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Text.UI.Wpf, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.TextManager.Interop, Version=7.1.40304.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.TextManager.Interop.10.0, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.TextManager.Interop.11.0, Version=11.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.TextManager.Interop.12.0, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.TextManager.Interop.8.0, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.TextTemplating.Interfaces.10.0, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=10.0.30319.1
    Microsoft.VisualStudio.TextTemplating.Interfaces.11.0, Version=11.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=11.0.50727.1
    Microsoft.VisualStudio.TextTemplating.VSHost.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Threading, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.51005.1
    Microsoft.VisualStudio.Toolbox, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Toolbox.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Utilities, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Utilities.Internal, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.51.52448
    Microsoft.VisualStudio.Validation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.50612.1
    Microsoft.VisualStudio.VCProjectEngine, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.VirtualTreeGrid, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.VSHelp, Version=7.0.3300.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.VSHelp80, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    Microsoft.VisualStudio.VsInteractiveWindow, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=1.3.1.60616
    Microsoft.VisualStudio.Web, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Web.Application, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.Web.Azure.Publish, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.Web.BrowserLink.12.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.Web.Editors.Razor.4_0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=null, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.Web.Exports, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Web.Extensions, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.Web.Extensions.Common, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.Web.HTML, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Web.HTML.Implementation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Web.HTML.Razor.Implementation.Shims.2_0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.60626.0
    Microsoft.VisualStudio.Web.HTML.Razor.Implementation.Shims.3_0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.60626.0
    Microsoft.VisualStudio.Web.Internal.Contracts, Version=2.3.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=2.3.61107.0
    Microsoft.VisualStudio.Web.Mvc.4.0, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=4.0.51102.0
    Microsoft.VisualStudio.Web.PackageManagement, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.Web.PageInspector.DataObjects, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.Web.PageInspector.Package, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.Web.PasteJson, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.Web.Project, Version=2.3.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=2.3.61107.0
    Microsoft.VisualStudio.Web.ProjectSystem, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.Web.Publish, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.Web.Publish.Contracts, Version=2.3.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=2.3.61107.0
    Microsoft.VisualStudio.Web.Publish.Core.Contracts, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.Web.PublishContract, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.Web.WindowsAzure.Contracts, Version=2.3.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=2.3.61107.0
    Microsoft.VisualStudio.Web.WindowsAzure.Explorer, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.Web.WindowsAzure.Impl, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.VisualStudio.WebClient.Diagnostics.HtmlToolHost, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.WebClient.Diagnostics.HtmlToolHost.resources, Version=14.0.0.0, Culture=en, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.WebClient.Diagnostics.Services, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VisualStudio.WindowsAzure.CommonAzureTools.Contracts, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.0.10930.1602
    Microsoft.VisualStudio.WindowsAzure.CommonAzureTools.Contracts.1.1, Version=1.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.1.20507.1601
    Microsoft.VisualStudio.WindowsAzure.CommonAzureTools.Contracts.1.2, Version=1.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.2.20710.1601
    Microsoft.VisualStudio.WindowsAzure.CommonAzureTools.Contracts.1.3, Version=1.3.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.3.21031.1601
    Microsoft.VisualStudio.WindowsAzure.CommonAzureTools.Contracts.1.4, Version=1.4.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.4.30416.1601
    Microsoft.VisualStudio.WindowsAzure.CommonAzureTools.Contracts.1.5, Version=1.5.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.5.30805.2
    Microsoft.VisualStudio.WindowsAzure.CommonAzureTools.Contracts.1.6, Version=1.6.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.6.30818.2
    Microsoft.VisualStudio.WindowsAzure.CommonAzureTools.Contracts.1.7, Version=1.7.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.7.40302.4
    Microsoft.VisualStudio.WindowsAzure.CommonAzureTools.Contracts.1.8, Version=1.8.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.8.40521.1
    Microsoft.VisualStudio.WindowsAzure.CommonAzureTools.Contracts.Internal, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.0.40521.1
    Microsoft.VisualStudio.WindowsAzure.CommonAzureTools.Package, Version=1.8.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.8.40521.1
    Microsoft.VisualStudio.WindowsAzure.Services.Common, Version=1.4.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.4.30611.1601
    Microsoft.VisualStudio.WindowsAzure.Services.Contracts.1.0, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.4.30611.1601
    Microsoft.VisualStudio.WindowsAzure.Services.Extensions, Version=1.4.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.4.30611.1601
    Microsoft.VisualStudio.WindowsAzure.Services.Extensions.MobileServicesHandlers, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.WindowsAzure.Services.Package, Version=1.4.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.4.30611.1601
    Microsoft.VisualStudio.WindowsAzure.Services.WebExtensions, Version=1.4.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.4.30611.1601
    Microsoft.VisualStudio.WPFFlavor, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.XmlEditor, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudio.XsdDesignerPackage, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VisualStudioTools.VSTestHost.14.0, Version=14.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.1.0
    Microsoft.VSDesigner, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VSDesigner.BootstrapPackage, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VSDesigner.ExceptionAssistant.SmartTag, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.VSDesigner.WCF, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VsHub.Client, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VsHub.Utility, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.VsSDK.ExtensibilityProjectFlavor, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.Web.Core, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.Web.Design.Client, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23010.0
    Microsoft.Web.Editor, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.1.11107.0
    Microsoft.Windows.Design.Extensibility, Version=4.3.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.Windows.Design.Interaction, Version=4.3.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    Microsoft.Windows.Simulator.Client, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.WizardFramework, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.WizardFrameworkVS, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    Microsoft.XmlEditor, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.23107.0
    mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, FileVersion=6.0.4.17603
    NuGet.Commands, Version=3.4.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=3.4.4.1321
    NuGet.Configuration, Version=3.4.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=3.4.4.1321
    NuGet.Console, Version=3.4.4.1321, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=3.4.4.1321
    NuGet.Console.Types, Version=3.4.4.1321, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=3.4.4.1321
    NuGet.Core, Version=2.12.0.815, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=(null)
    NuGet.Credentials, Version=3.4.4.1321, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=3.4.4.1321
    NuGet.Frameworks, Version=3.4.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=3.4.4.1321
    NuGet.Logging, Version=3.4.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=3.4.4.1321
    Nuget.PackageIndex, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.0.31.0
    Nuget.PackageIndex.VisualStudio, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.0.31.0
    NuGet.PackageManagement, Version=3.4.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=3.4.4.1321
    NuGet.PackageManagement.UI, Version=3.4.4.1321, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=3.4.4.1321
    NuGet.PackageManagement.VisualStudio, Version=3.4.4.1321, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=3.4.4.1321
    NuGet.Packaging, Version=3.4.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=3.4.4.1321
    NuGet.Packaging.Core.Types, Version=3.4.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=3.4.4.1321
    NuGet.ProjectManagement, Version=3.4.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=3.4.4.1321
    NuGet.ProjectModel, Version=3.4.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=3.4.4.1321
    NuGet.Protocol.Core.Types, Version=3.4.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=3.4.4.1321
    NuGet.Protocol.Core.v2, Version=3.4.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=3.4.4.1321
    NuGet.Protocol.VisualStudio, Version=3.4.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=3.4.4.1321
    NuGet.Resolver, Version=3.4.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=3.4.4.1321
    NuGet.Tools, Version=3.4.4.1321, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=3.4.4.1321
    NuGet.Versioning, Version=3.4.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=3.4.4.1321
    NuGet.VisualStudio, Version=3.4.4.1321, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=3.4.4.1321
    NuGet.VisualStudio.Implementation, Version=3.4.4.1321, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=3.4.4.1321
    NuGetConsole.Host.PowerShell, Version=3.4.4.1321, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=3.4.4.1321
    NuGetConsole.Host.PowerShellProvider, Version=3.4.4.1321, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=3.4.4.1321
    PreEmptive.Analytics.Aggregator.Client, Version=1.2.5134.1, Culture=neutral, PublicKeyToken=b0f717fa0dcc6e86, FileVersion=(null)
    PreEmptive.Analytics.Aggregator.Common, Version=1.2.5134.1, Culture=neutral, PublicKeyToken=b0f717fa0dcc6e86, FileVersion=(null)
    PreEmptive.Analytics.Aggregator.VSPackage.14.0, Version=1.2.5134.1, Culture=neutral, PublicKeyToken=b0f717fa0dcc6e86, FileVersion=(null)
    PreEmptive.Analytics.Aggregator.VSPackage.TeamFoundation.14.0, Version=1.2.5134.1, Culture=neutral, PublicKeyToken=b0f717fa0dcc6e86, FileVersion=(null)
    PresentationCore, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=4.6.1584.0
    PresentationFramework, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=4.6.1584.0
    PresentationFramework.Aero2, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=4.6.1584.0
    PresentationFramework-SystemCore, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    PresentationFramework-SystemData, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    PresentationFramework-SystemXml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    PresentationFramework-SystemXmlLinq, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    SMDiagnostics, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    StanCore, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    StanPackage, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    stdole, Version=7.0.3300.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    System.Activities.Presentation, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=4.6.1584.0
    System.Collections, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Collections.Concurrent, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Collections.Immutable, Version=1.1.37.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.23123.00
    System.ComponentModel.Composition, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    System.ComponentModel.DataAnnotations, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=4.6.1584.0
    System.Composition.AttributedModel, Version=1.0.27.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.0.27.0
    System.Composition.Runtime, Version=1.0.27.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.0.27.0
    System.Configuration, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    System.Data.Entity, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    System.Data.Entity.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Diagnostics.Debug, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Diagnostics.Tools, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Diagnostics.Tracing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Drawing.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Dynamic, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.EnterpriseServices, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Globalization, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.IdentityModel, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    System.IO, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.IO.FileSystem, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.23123.0
    System.Linq, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Linq.Expressions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Net.Http, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Net.Http.WebRequest, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Numerics, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    System.Reflection, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Reflection.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Reflection.Metadata, Version=1.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.23907.00
    System.Reflection.Primitives, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Runtime.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Runtime.Handles, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Runtime.InteropServices, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Runtime.Remoting, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    System.Runtime.Serialization, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    System.Runtime.Serialization.Formatters.Soap, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Security, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.ServiceModel, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    System.ServiceModel.Internals, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=4.6.1584.0
    System.Text.Encoding, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Text.Encoding.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Threading, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Threading.Tasks, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Threading.Tasks.Dataflow, Version=4.5.24.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.5.24.0
    System.Transactions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    System.Web, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=4.6.1584.0
    System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=4.6.1584.0
    System.Web.Razor, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=2.0.20715.0
    System.Web.Razor, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=3.0.20826.0
    System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    System.Windows.Forms.VisualStudio.11.0, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=14.0.23107.0
    System.Xaml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    System.Xml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    System.Xml.Linq, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, FileVersion=4.6.1584.0
    TestSccPackage, Version=1.0.0.2014, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.0.0.00
    TMLanguage, Version=1.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.1.60520.2
    TMParser, Version=1.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=1.1.60520.2
    UIAutomationProvider, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=4.6.1584.0
    UIAutomationTypes, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=4.6.1584.0
    VSDebugCoreUI, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    VsDebugPresentationPackage, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    VsGraphicsDebuggerPkg, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1
    VSLangProj, Version=7.0.3300.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    VSLangProj110, Version=11.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    VSLangProj2, Version=7.0.5000.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    VSLangProj80, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    VsWebSite.Interop, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=(null)
    WindowsBase, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=4.6.1584.0
    WindowsFormsIntegration, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, FileVersion=4.6.1584.0
    WpfXamlDiagnosticsTap, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, FileVersion=14.0.25420.1

Analysis Level: NodeLsNone

IntelliSense Completion Only Tab or Enter to Commit: True
```